ui/pages/settings/networks-tab/networks-tab.js

```diff
@@ -50,6 +54,7 @@ const NetworksTab = ({ addNewNetwork }) => {
       chainId: rpc.chainId,
       ticker: rpc.ticker,
       blockExplorerUrl: rpc.rpcPrefs?.blockExplorerUrl || '',
+      userIsCurrentlyOnATestNet: TEST_CHAINS.includes(rpc.chainId),
```

Comment:
Because `frequentRpcListDetail` can have more than one network, then `userIsCurrentlyOnATestNet` may be a misleading name. To say that a "user is currently on" a network implies that the network is selected. So when reading other files that reference this property, it might seem like this is a value that indicates whether the current network is a test network.

I suggest renaming this to `isATestNetwork` to communicate the meaning of this property correctly

Reply:
@danjm Fixed in: 2179697b44f506d161e5258c52789f632ae8bda8